### DISCUSSION TITLE
Dracula on flatpaks

### DISCUSSION THREAD

**Discussion by syds-git** [dracula/dracula-theme#587] [Q&A] [answered]
I can't get my flatpaks to use Dracula. Does anyone know how to or to create a flatpak version of Dracula?

**Comment by trytonlux** [comment, marked as answer]
For a GTK theme, I've created a flatpak to use Ant-Dracula (https://github.com/tryton-vanmeer/Flatpak-Gtk3themes) in the past. It could serve as a reference.

For other themes: A flatpak has it's XDG data and config directories set in `~/.var/app/<app-id>` (For a  system installed app, it's `/var/app/<app-id>` I believe).

**Reply by trytonlux** [reply]
Came across a blog post recently  that goes over it in more detail, including a local private flatpak extension.

https://blog.tingping.se/2023/04/20/Themes-and-Flatpak.html
